utils/rpms/daos_limits.conf

```diff
@@ -0,0 +1,16 @@
+# /etc/security/limits.d/daos_limits.conf
+#
+# This file sets DAOS resource limits for users logged in via PAM.
+# It does not affect resource limits of the system services, which
+# are specified in the DAOS systemd service files:
+# * /usr/lib/systemd/system/daos_server.service
+# * /usr/lib/systemd/system/daos_agent.service
+#
+# Each line describes a limit for a user in the form:
+#
+# <domain>        <type>  <item>  <value>
+
+* soft memlock unlimited
+* hard memlock unlimited
```

Comment:
It's slightly concerning that we are raising this limit for every{one,thing} on a system where this is installed rather the much more fine-grained limit increase that is done for just the daos processes by the systemd unit file.

I'd like to [understand the use-case](https://daosio.atlassian.net/browse/DAOS-11247?focusedCommentId=104170) for this a bit better.

Reply:
see my update in https://daosio.atlassian.net/browse/DAOS-11247